README.md

```diff
@@ -75,7 +86,11 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+
+var pizza = {temperature: 70, bake: function bake(temp) {
+ return this.temperature = 300;
```

Comment:
No need for the ```return``` here. You're just changing the variable within the object, so return is not necessary